### PR TITLE
DL4J Docs Update 2.15.2019

### PR DESCRIPTION
**Pull request by ejunprung**

### CODE REVIEW

**Comment by ejunprung** [comment]
@crockpotveggies Check it out. Autogen seems to work perfectly for me now.

File: docs/_latest/deeplearning4j-nn-convolutional.md [hunk 1]
```diff
@@ -12,387 +12,4 @@ Each layer in a neural network configuration represents a unit of hidden units.
...
-public Cropping3D build() 
-```
-
-
-- param cropping Cropping amount, must be length 3 or 6 array, i.e. either
-crop depth, crop height, crop width or
-crop left depth, crop right depth, crop left height, crop right height, crop left width,
-crop right width
-
+{{autogenerated}}
```
Comment (crockpotveggies) [inline docs/_latest/deeplearning4j-nn-convolutional.md]: Problem starts here.

File: docs/_latest/deeplearning4j-nn-early-stopping.md [hunk 2]
```diff
@@ -80,180 +80,4 @@ Note that `EarlyStoppingParallelTrainer` doesn't support all of the functionalit
...
-
-
-
----
-
-### ScoreImprovementEpochTerminationCondition
-<span style="float:right;"> [[source]](https://github.com/deeplearning4j/deeplearning4j/tree/master/deeplearning4j/deeplearning4j-nn/src/main/java/org/deeplearning4j/earlystopping/termination/ScoreImprovementEpochTerminationCondition.java) </span>
-
-Terminate training if best model score does not improve for N epochs
+{{autogenerated}}
```
Comment (crockpotveggies) [inline docs/_latest/deeplearning4j-nn-early-stopping.md]: Same problem exists here.

File: docs/_latest/deeplearning4j-nn-recurrent.md [hunk 3]
```diff
@@ -352,195 +352,4 @@ Note that in the case of training data that contains time series of different le
...
-### SimpleRnn
-<span style="float:right;"> [[source]](https://github.com/deeplearning4j/deeplearning4j/tree/master/deeplearning4j/deeplearning4j-nn/src/main/java/org/deeplearning4j/nn/conf/layers/recurrent/SimpleRnn.java) </span>
-
-Simple RNN - aka "vanilla" RNN is the simplest type of recurrent neural network layer.
-
-Note that other architectures (LSTM, etc) are usually much more effective, especially for longer time series;
-however SimpleRnn is very fast to compute, and hence may be considered where the length of the temporal dependencies
-in the dataset are only a few steps long.
-
+{{autogenerated}}
```
Comment (crockpotveggies) [inline docs/_latest/deeplearning4j-nn-recurrent.md]: Same problem exists here.

File: docs/_latest/deeplearning4j-nn-vertices.md [hunk 4]
```diff
@@ -12,162 +12,4 @@ In Eclipse Deeplearning4j a vertex is a type of layer that acts as a node in a `
...
-##### setBackpropGradientsViewArray 
-```java
-public void setBackpropGradientsViewArray(INDArray backpropGradientsViewArray) 
-```
-
-
-Gets the current mask array from the provided input
-- return The mask or null, if no input was provided
-
+{{autogenerated}}
```
Comment (crockpotveggies) [inline docs/_latest/deeplearning4j-nn-vertices.md]: Same problem exists here.

**Comment by ejunprung** [comment]
@crockpotveggies I'm confused. So {{autogenerated}} should automatically insert the Java classes, right? 

If so, it might be an issue with the auto gen script. Does it happen to be working on your computer? It seemed to run fine on my Macbook but I'm not sure of the expected behavior.

**Comment by crockpotveggies** [comment]
One possible explanation is that the java classes have moved locations. Not sure without looking at it further, can you copy/paste the output from the gen docs script and send it to me?

**Comment by ejunprung** [comment]
https://gist.github.com/ejunprung/2d74016bc1332bf6e61275a66bff3e6f

**Comment by crockpotveggies** [comment]
Yea there's an error in there I'm guessing a package changed.

Best,
Justin
